run.sh

```diff
@@ -52,20 +52,8 @@ echo "Prerequisites look good!"
 echo "Loading .token..."
 tg_token=$(cat .token)
 
-# Load cookies from .cookies file and concat into string
-echo "Loading .cookies..."
-bing_cookies=""
-while read line; do
-  if [ -n "$line" ]; then
-    bing_cookies="$bing_cookies'${line}' "
-  fi
-done < <(grep . .cookies)
-
-# Remove trailing space
-bing_cookies="${bing_cookies% }"
-
 # Building and running
-python_cmd="${python_bin} tg.py '$tg_token' $bing_cookies"
+python_cmd="${python_bin} -u tg.py '$tg_token'"
```

Comment:
why -u here?

Reply:
https://github.com/yihong0618/tg_bing_dalle/blob/42839e5f012b385c02e54cb755cb0967a232ddca/entrypoint.sh#L28C3-L28C55

Disable buffering to see the output immediately